src/test/java/core/basesyntax/strategy/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static final String KEY = "apple";
+    private static OperationHandler operationHandler;
+    private static FruitTransaction fruitTransaction;
+    private static FruitDao fruitDao;
+
+    @BeforeClass
+    public static void init() {
+        operationHandler = new BalanceOperationHandler();
+        fruitTransaction = new FruitTransaction();
+        fruitDao = new FruitDaoImpl();
+        fruitTransaction.setFruit(KEY);
+        fruitTransaction.setQuantity(20);
```

Comment:
(Suggestion only) I think it'll be better if you transfer these values right to the constructor on 22 line